x-pack/test/functional/es_archives/dashboard/drilldowns/mappings.json

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
Did you try reusing existing archived data?  Wondering why you needed to create a new archive (don't want to duplicate data that adds to repo size, unless there is a reason)

Reply:
@stacey-gammon, I decided to prepare convenient data upfront. there are:

2 dashboards: 
1. with pie chart to test value click action
2. with area chart to test range select action. - **this one contains preconfigured drilldown** - to make sure we don't break drilldowns created in 7.8 in future versions.